src/Chapter1.hs

```diff
@@ -641,7 +659,10 @@ You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
 
-firstDigit n = error "firstDigit: Not implemented!"
+firstDigit :: Integral t => t -> t
+firstDigit n 
+  | abs n < 10 = abs n
+  | otherwise = firstDigit ((abs n) `div` 10)
```

Comment:
That is right 💯 

You can see that `abs n` is reused several times in this function. This usually means that you can move it into the `where` case and reuse everywhere 👍🏼